server/config/config.go

```diff
@@ -1181,6 +1181,7 @@ type DashboardConfig struct {
 	TiDBKeyPath      string `toml:"tidb-key-path" json:"tidb_key_path"`
 	PublicPathPrefix string `toml:"public-path-prefix" json:"public_path_prefix"`
 	InternalProxy    bool   `toml:"internal-proxy" json:"internal_proxy"`
+	DisableTelemetry bool   `toml:"disable-telemetry" json:"disable_telemetry"`
```

Comment:
Does the config option need more comments?

Reply:
There are instructions in the code of `tidb-dashboard` and `config.toml`.